.github/workflows/pr-labels.yml

```diff
@@ -0,0 +1,31 @@
+name: Pull Request Label
+
+on: pull_request
+
+jobs:
+  setup-label:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Setup
+        uses: ./tools/github-actions/setup
+        with:
+          enable-build-cache: 'false'
+      - run: |
+          LABELS=$(yarn pr-labels $(git log -1 --pretty=%B))
```

Comment:
is working with multi message commit (multiple `-m` or multiple lines)?
Maybe it could be surrounded by `"` to be given as 1 parameter?

Reply:
didn't test I can check :)